assets/data/aspects/drauvenSkillDeck.json

```diff
@@ -21,240 +21,273 @@
+	"effects": [ "drauven_pc_channelFireball" ],
+	"boostType": "ABILITY_CLASS",
 	"legacyAbilityChoice": true,
 	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
-	"showInTooltip": true,
-	"info": { "abilityDeckUsage": "upgrade", "upgradeForAbility": "drauvenDeck_warrior_concussed", "abilityDeckIcon": "bloodrage" }
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_mystic_fireshot",
+		"abilityDeckIcon": "fireleash"
+	}
 },
 {
 	"modId": "wildermyth-drauven-pcs",
-	"id": "drauvenDeck_warrior_outrage",
-	"effects": [ "drauven_pc_outrage" ],
+	"id": "drauvenDeck_warrior_bloodedFerocity",
+	"effects": [ "drauven_pc_bloodedFerocity" ],
 	"boostType": "ABILITY_CLASS",
 	"legacyAbilityChoice": true,
 	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
-	"showInTooltip": true,
-	"info": { "abilityDeckUsage": "warrior", "abilityDeckIcon": "soulHarvest" }
+	"info": { "abilityDeckUsage": "warrior", "abilityDeckIcon": "ignite" }
 },
 {
 	"modId": "wildermyth-drauven-pcs",
-	"id": "drauvenDeck_warrior_outrage_upgrade",
+	"id": "drauvenDeck_warrior_bloodedFerocity_upgrade",
+	"effects": [ "drauvenDeck_bloodedFerocity_upgrade" ],
 	"legacyAbilityChoice": true,
 	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
-	"showInTooltip": true,
-	"info": { "abilityDeckUsage": "upgrade", "upgradeForAbility": "drauvenDeck_warrior_outrage", "abilityDeckIcon": "soulHarvest" }
+	"info": {
+		"abilityDeckUsage": "upgrade",
+		"upgradeForAbility": "drauvenDeck_warrior_bloodedFerocity",
+		"abilityDeckIcon": "ignite"
+	}
 },
 {
 	"modId": "wildermyth-drauven-pcs",
-	"id": "drauvenDeck_hunter_preparedShot",
-	"effects": [ "drauven_pc_preparedShot" ],
+	"id": "drauvenDeck_warrior_concussed",
+	"effects": [ "drauven_pc_concussed" ],
 	"boostType": "ABILITY_CLASS",
 	"legacyAbilityChoice": true,
 	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
-	"info": { "abilityDeckUsage": "hunter", "abilityDeckIcon": "ambush" }
+	"info": { "abilityDeckUsage": "warrior", "abilityDeckIcon": "bloodrage" }
 },
 {
 	"modId": "wildermyth-drauven-pcs",
-	"id": "drauvenDeck_hunter_preparedShot_upgrade",
+	"id": "drauvenDeck_warrior_concussed_upgrade",
 	"legacyAbilityChoice": true,
 	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
+	"showInTooltip": true,
 	"info": {
 		"abilityDeckUsage": "upgrade",
-		"upgradeForAbility": "drauvenDeck_hunter_preparedShot",
-		"abilityDeckIcon": "ambush"
+		"upgradeForAbility": "drauvenDeck_warrior_concussed",
+		"abilityDeckIcon": "bloodrage"
 	}
 },
 {
 	"modId": "wildermyth-drauven-pcs",
-	"id": "drauvenDeck_hunter_quickblade",
-	"effects": [ "drauven_pc_quickblade" ],
+	"id": "drauvenDeck_warrior_outrage",
+	"effects": [ "drauven_pc_outrage" ],
 	"boostType": "ABILITY_CLASS",
 	"legacyAbilityChoice": true,
 	"abilityRequiresOneAspectOf": [ "drauven_pc" ],
-	"info": { "abilityDeckUsage": "hunter", "abilityDeckIcon": "thornfang" }
+	"showInTooltip": true,
+	"info": { "abilityDeckUsage": "warrior", "abilityDeckIcon": "soulHarvest" }
```

Comment:
🤔 To remove Outrage from the Warrior deck, I think we would just need to remove the `abilityDeckUsage`, aye?  That should keep Outrage as a skill for anyone who already has it in Legacy or active games, but remove the ability for new characters to get it. I think that method of deprecation more or less follows the general philosophy for other parts of the game... What do you think?